app/main.py

```diff
@@ -1,44 +1,70 @@
 import time
+import asyncio
 
 from iot.devices import HueLightDevice, SmartSpeakerDevice, SmartToiletDevice
 from iot.message import Message, MessageType
 from iot.service import IOTService
 
 
-def main() -> None:
+async def main() -> None:
     # create an IOT service
     service = IOTService()
 
     # create and register a few devices
     hue_light = HueLightDevice()
     speaker = SmartSpeakerDevice()
     toilet = SmartToiletDevice()
+
     hue_light_id = service.register_device(hue_light)
     speaker_id = service.register_device(speaker)
     toilet_id = service.register_device(toilet)
 
```

Comment:
You need to use ```await asyncio.gather``` here for register

Reply:
https://github.com/mate-academy/py-asyncio-iot/pull/240#discussion_r1434126401
So I do it